src/main/java/seedu/address/model/Model.java

```diff
@@ -57,6 +57,16 @@ public interface Model {
      */
     boolean hasPerson(Person person);
 
+    /**
```

Comment:
This pattern is good, allows for easy extensions without messing with existing features i.e Open/Closed Principle